doc: Explicitly state that there are some incompatible rules CY-4284 (#1485)
codacy/codacy-eslint: README.md

5.  Publish your docker locally as normal: `docker build -t codacy-eslint:local .`

## Limitations

### Incompatible rules

> There are some ESLint rules that will be ignored when running this docker container. For more details on the ignored
> rules, you can check `blacklistRegexes` defined at [blacklist.ts](src/blacklist.ts).
> 
> Usually those rules need internet connection, and/or to check `node_modules`, and would not run successfully
> on our docker container envorionment.

Reviewer: ```suggestion
There are some ESLint rules that will be ignored when running this Docker container. For more details on the ignored
rules, check `blacklistRegexes` defined at [blacklist.ts](src/blacklist.ts).

Usually, these rules need an Internet connection and/or to check `node_modules`, and would not run successfully
on our Docker container environment.
```
